cmd/habitat/node/node.go

```diff
@@ -35,6 +40,10 @@ type Node struct {
 	P2PNode        *p2p.Node
 	ReverseProxy   *proxy.Server
 	DataProxy      *dataproxy.DataProxy
+	FS             *fs.FS
```

Comment:
hmm it feels weird that both FS and IPFSClient are top level for node. is ipfs used outside of FS?